lib/bundles/inspec-supermarket/target.rb

```diff
@@ -17,7 +17,7 @@ def self.resolve(target, opts = {})
       return nil unless URI(target).scheme == 'supermarket'
```

Comment:
should we change the
```
def self.resolve(target, opts = {})
```
to 
```
def self.resolve(opts = {})
```
since we having a hash only. This would allow us to start with `:url` and resolve it from there?